plugins/widget/plugin.js

```diff
@@ -2701,6 +2754,22 @@
 				var attrs = element.attributes,
 					widget, widgetElement;
 
+				// Reset initial and trailing space by replacing &nbsp; with white space (#605).
+				if ( 'data-cke-widget-white-space' in attrs ) {
+					var firstTextNode = getFirstTextNode( element ),
```

Comment:
The main point of adding 'data-cke-white-space-first|last' data attributes to specific elements was to enable the easy way to find them later without a need to iterate over whole widget structure. However, to find them you still need to use `find` method same as in `getFirstTextNode`/`getLastTextNode` so it doesn't make a difference. Lets leave it as it is.